src/apel_plugin.py

```diff
@@ -492,10 +492,7 @@ async def get_token(config):
     return token
 
 
-async def sign_msg(config, msg):
-    client_cert = config["authentication"].get("client_cert")
-    client_key = config["authentication"].get("client_key")
-
+async def sign_msg(client_cert, client_key, msg):
```

Comment:
`client_key` is unused and function doesn't return anything?

Reply:
Also, function doesn't need to be `async`.

Reply:
`client_key` is opened to get the private key, which is used to sign the message, and the function returns the signed message
not sure about `async`, I will test it without

Reply:
Sorry, my fault. Github only showed me the first two lines of this function and I didn't see the expand-icon because of the sun shining on my screen :facepalm:

Reply:
ok, I'm planning to remove unnecessary `async` functions in a separate pull request